src/documentation/markdown/2018/lar-data-fields.md

```diff
@@ -400,7 +400,7 @@
   - 48%
   - 49%
   - 50%-60%
-  - >60%
+  - 60%
```

Comment:
Can we please double check that this bin is exclusively for 60%? 60% seems to already be accounted for in the previous bin.

Reply:
Has been updated via stand up feedback - dated: 11-4-2022.